src/olympia/addons/tests/test_views.py

```diff
@@ -3158,7 +3158,26 @@ def test_compatibility_forbidden_range_android(self):
         }
 
         self.make_addon_promoted(self.addon, RECOMMENDED, approve_version=True)
-        response = self.request(compatibility={'android': {'min': '48.0', 'max': '*'}})
+        response = self.request(
+            compatibility={'android': {'min': '119.0a1', 'max': '*'}}
```

Comment:
I see you've changed this request so it's different than the non-recommended request.  Does it work differently for recommended add-ons too now?

Reply:
No, you're right, I shouldn't have made that change. Fixed in c50baf291088ffb80c924ad91c24096b3d14b06a